src/vision/frc_vision_final.py

```diff
@@ -13,6 +13,8 @@
 import time
 from networktables import NetworkTables
 
+angleThreshold = 40
```

Comment:
The name doesn't really follow the `SNAKE_CASE` convention set elsewhere in the file

Reply:
this is also the case with some other int values like pointer, maybe change those while you're at it for consistency's sake